client/lib/push-notifications/index.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * External dependencies
+ */
+import Emitter from 'lib/mixins/emitter';
+import debugFactory from 'debug';
+import moment from 'moment';
+import store from 'store';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import wpcom from 'lib/wp';
+
+const daysBeforeForcingRegistrationRefresh = 15,
```

Comment:
I think (but I'm not sure) it's conventional to specify these like we do in PHP: all-caps w/ underscores between words.